### ISSUE TITLE
Create a guide: Responsible use of images on HfLA websites

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#707]
### Overview
We need a guide for image use on on our open source projects so that we can be in compliance and use best practices.

### Action Items
- [ ] PM
   - [x]  fill out rest of issue to dos
- [ ] Research and write about
   -  [x] Where to find image
   -  [x] How to do attribution
   - [x] storing images
   - [x] image formats
   - [x] sizing best practices
- [ ] Discuss and assign to  HfLA developers + designers for feedback  
  - [ ] Give Feedback on this: [Guide](https://docs.google.com/document/d/17AbUtj9aIA8G88O8AVJnyL1jz4cNQ3x740cmzO-2yzk/edit)
- [ ]  Open Issue on UI/UX Communities of Practice repo for feedback 
- [ ] UI/UX
  - [ ] mock up page
  - [ ] This guide issue needs to be updated on our [Guide Tracking sheet](https://docs.google.com/spreadsheets/d/1omYkR_WeploNuOkF3Pqsg6DhIdej5SS2EBbG9FRefbA/edit?usp=sharing)

### Resources/Instructions
[Wiki](https://github.com/hackforla/website/wiki/Resources)
[Guide](https://docs.google.com/document/d/17AbUtj9aIA8G88O8AVJnyL1jz4cNQ3x740cmzO-2yzk/edit)
[Unsplash](https://unsplash.com/)
[Pexels](https://pexels.com/)
[the noun project](https://thenounproject.com/)
[Font Awesome](https://fontawesome.com/)
[to create book covers](https://edit.org/book-covers)
[Hack for LA Social Media Icons](https://drive.google.com/drive/folders/1xuyqklBJo5i7ReTTFGsCmOKZ4ksCYJH4)
[Credits page](https://www.hackforla.org/credits)
[Creative Commons Image List on Google](https://www.google.com/search?q=dog&tbm=isch&hl=en&tbs=il:cl&sa=X&ved=0CAAQ1vwEahcKEwj44fD90oDuAhUAAAAAHQAAAAAQAg&biw=2314&bih=881) <- replace dog with what you are searching for

**Comment by ahdithebomb** [comment]
Chatted with @jbubar & @daniellen00 this week to gather initial information. We will conduct additional research to finalize the following sections.
- [ ] Image Formats
- [ ] Sizing Best Practices

Overall first pass of the guide to be completed in 2 weeks with mock-ups.

**Comment by ahdithebomb** [comment]
Linke to the guide 
[Guide](https://docs.google.com/document/d/17AbUtj9aIA8G88O8AVJnyL1jz4cNQ3x740cmzO-2yzk/edit)

**Comment by ahdithebomb** [comment]
Progress: Got directions from @ExperimentsInHonesty on what overall improvements need to be made. 
Blocks: N/A
Availability: I Will coordinate with @daniellen00 and make necessary updates. 
ETA: End of this upcoming week.

**Comment by ahdithebomb** [comment]
The same updates apply as of Sunday.

**Comment by ahdithebomb** [comment]
Progress:  Added some screenshots & @daniellen00 added some more commentary about image types. 
Blocks: Not sure where to place the md instructions in the team wiki. 
Availability: I Will coordinate with @daniellen00 and make necessary updates.
ETA: End of this upcoming week.

**Comment by ahdithebomb** [comment]
Progress: I added the MD instructions to the wiki. 
Blocks: Need to coordinate with Cynthia to finalize last section, and we need to add some prototypes. 
Availability: I Will coordinate with @daniellen00 and make necessary updates.
ETA: End of this upcoming week.

**Comment by ahdithebomb** [comment]
Progress: The first draft is mostly complete with mockups/images. 
Blocks: Waiting on feedback from @ExperimentsInHonesty for our first pass.
Availability: I Will coordinate with @daniellen00 and make necessary updates.
ETA: End of this upcoming week, I will also ping @jbubar to get more insights into file compression.

**Comment by ahdithebomb** [comment]
Progress:  Same as last week 
Blocks: Need team's feedback to determine whether the guide makes/doesn't make sense. 
Availability: I Will coordinate with @daniellen00 and make necessary updates.
ETA: TBD

**Comment by ahdithebomb** [comment]
Progress: Same as last week
Blocks: We need more assignees to take a look at this!
Availability: Depending on when assignees have time to provide feedback.
ETA: TBD

**Comment by ahdithebomb** [comment]
Progress: Same as last week
Blocks: We need more assignees to take a look at this!
Availability: Depending on when assignees have time to provide feedback.
ETA: TBD

**Comment by tarang100** [comment]
@daniellex0 we have had quite a few comments. We should figure out who will be responsible for the changes that people have suggested

**Comment by qiqicodes** [comment]
Danielle and Candice to discuss

**Comment by ruqpyL2** [comment]
Update 3/27
Progress: Reorg, smoothing, and adding more content (img compression, ELI5, more practice pointers). Maybe halfway through.
Blockers: Danielle and I both have a ton of job-work.
Availability: Enough to get it done
ETA: I think I'll have a draft by Friday next week, then it'll be Danielle's term

**Comment by daniellex0** [comment]
Update 4/4
Progress: @ruqpyL2 added a lot of missing documentation about compressing images, scaling, responsiveness, image format conversion, and general UX writing edits. (Thank you Candace!). We met and reviewed/finalized the edits together, and Candace then shaved off some extra text/polished overall guide. Next steps: verify some last points and submit.
Blockers: Busy
Availability: Enough to get it done
ETA: This week

**Comment by ExperimentsInHonesty** [comment]
@ruqpyL2 @daniellex0 Please put the guide in this format: https://docs.google.com/document/d/1-8gmeC-wnfM8C8fVvmTP1BLA2WfNjKH_XJrHB1OAV6Q/edit#heading=h.8kycsl8g2y4x

**Comment by ExperimentsInHonesty** [comment]
@daniellex0 and @ExperimentsInHonesty will review during weekly meeting

**Comment by Olivia-Chiong** [comment]
hey @daniellex0 and @ruqpyL2 is someone reformatting this guide? Could I get an update please?

Please update the following

- **Progress:** _How much progress have you made on the issue?_
- **Blocks:** _Are you facing any blockers?_
- **Availability:** _How much time do you have to work on it this week?_
- **ETA:** _When do you think it will be completed?_

**Comment by daniellex0** [comment]
This is actually finished! It just needs final review by @ExperimentsInHonesty or whoever is interested/qualified to provide final review (there have already been lots of reviews before, and their feedback was incoporated):

https://docs.google.com/document/d/1w9EIJYe-HL5dVbTnp8dTiyOM_KDR9VH8RBM77bR30Co/edit

**Comment by daniellex0** [comment]
Progress: Guide was reviewed and approved- I just need to move around some of the sections per feedback and it will be ready.
Blocks: None
ETA: Tue

**Comment by SaashaG** [comment]
For devs: guide card Illustration and image credit info in Toolkit page Figma file (blue rectangle on bottom left): https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1548%3A1967 "

Illustration:
Link: https://storyset.com/illustration/image-upload/pana#default&hide=Plants&hide=simple
Attribute: <a href="https://storyset.com/online">Online illustrations by Storyset</a>

**Comment by fchan218** [comment]
<details><summary>Assignee, Labels, Project Board Placement, and Milestones for this issue in the Website Repo:</summary>
<p>
<img width="336" alt="Screenshot 2024-10-26 at 11 57 09 AM" src="https://github.com/user-attachments/assets/1f93a2cf-b057-4f6a-9774-65f91a791f24">
</p>
</details>

**Comment by fchan218** [comment]
### Files in this issue:

- [Guide: Responsible use of images on HfLA websites](https://docs.google.com/document/d/17AbUtj9aIA8G88O8AVJnyL1jz4cNQ3x740cmzO-2yzk)
   - This file will need to be moved to the knowledgebase-content Google Drive

- [Image Use on Websites Guide (New format)](https://docs.google.com/document/d/1w9EIJYe-HL5dVbTnp8dTiyOM_KDR9VH8RBM77bR30Co)
   - This file will need to be moved to the knowledgebase-content Google Drive

**Comment by fchan218** [comment]
@daniellex0 This guide issue is moving to the knowledgebase-content repo. If you would like to maintain edit access to the issue and its associated files, please fill out the [Google Drive Access Form](https://forms.gle/vnKRqprNYJUG5ET69).
